app/assets/javascripts/autocomplete.js.coffee

```diff
@@ -34,7 +34,7 @@
         suggestion.display
     source: mySource
     limit: 10
-  ).on("typeahead:selected typeahead:autocompleted", (event, suggestion, dataset) ->
```

Comment:
This behaviour is defined https://devdocs.io/jqueryui/autocomplete and https://github.com/twitter/typeahead.js/blob/master/doc/jquery_typeahead.md#custom-events

Reply:
Genre, Language and User/Group Access appear to be the only fields that use this javascript.

Reply:
:+1: